internal/bminventory/inventory.go

```diff
@@ -2843,6 +2846,14 @@ func (b *bareMetalInventory) uploadLogs(ctx context.Context, params installer.Up
 		return nil
 	}
 
+	if params.LogsType == string(models.LogsTypeController) {
```

Comment:
lets not make it the same way as hosts. I did it with hosts only for backward compatibility support.

Reply:
The only if we need is at the end to set flags after upload, all other code is the same.

Reply:
done.